specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -363,6 +363,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/locations/{location}/detailedusages": {
+      "get": {
+        "tags": [
+          "DetailedUsage"
+        ],
+        "operationId": "DetailedUsages_List",
+        "description": "Gets the current detailed (workspace, cluster level) usage information as well as limits for AML resources for given subscription and location.",

```

Comment:
>s limits for AML resources for given subscription and locati [](start = 104, length = 60)

I don't see any current value or limit info in the usage or response schema for this api ..

 the example for GET /usage has the following and I am expecting the new api response to have some of the info in this. Isn't it ?

 {
            "currentValue": 1,
            "limit": 600,
            "name": {
              "localizedValue": "Standard D Family Cluster Dedicated vCPUs",
              "value": "Standard D Family Cluster Dedicated vCPUs"
            },
            "unit": "Count"
          },